plugins/generator-1.20.4/neoforge-1.20.4/triggers/player_completes_advancement.java.ftl

```diff
@@ -8,7 +8,7 @@
 			"z": "event.getEntity().getZ()",
```

Comment:
AdvancementEvent is abstract and as per https://neoforged.net/news/20.2eventbus-changes/, this will crash the game.

This gives me doubt you have tested those triggers.

Please retest all triggers in open PRs and mark them ready for review once tested and cases like this one are fixed